DNN Platform/Tests/DotNetNuke.Tests.Core/Controllers/Portal/PortalControllerTests.cs

```diff
@@ -106,6 +109,9 @@ public void SetUp()
         public void TearDown()
         {
             PortalTemplateIO.ClearInstance();
+
+            // NOTE: not sure why, but disposing here causes other unrelated tests to fail
+            ////this.serviceProvider.Dispose();

```

Comment:
FYI, this was a weird issue I had to work around. I'm not sure what would have changed to cause this to start cropping up.